client/src/js/services/receipts/ReceiptModal.js

```diff
@@ -41,7 +41,9 @@ function ReceiptModal(Modal, Receipts) {
    * @param {String} uuid             Target invoice UUID
    * @param {Boolean} notifyCreated   Defines if a success message should be shown for entity creation
    */
-  function invoice(uuid, notifyCreated) {
+  async function invoice(uuid, notifyCreated) {
```

Comment:
Unfortunately, we can't use async/await on the client without some hacks, due to the way AngularJS's $digest loop works.  For reference, you can see [this post](https://medium.com/@alSkachkov/using-async-await-function-in-angular-1-5-babel-6-387f7c43948c),